libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -299,25 +311,25 @@ export class DropdownComponent
   }
 
   private showDropdown() {
-    if (this.state === openState.opening) {
-      this.state = openState.open;
+    if (this.state === OpenState.opening) {
+      this.state = OpenState.open;
       this.scrollItemIntoView(this.selectedIndex);
       this.changeDetectorRef.markForCheck();
     }
   }
 
-  close() {
+  closeDropdown() {
```

Comment:
Unfortunately `close()` is part of the public and published API, so shouldn't change (or at least have a very good reason to...)

Reply:
Noted 👍  (and changed back again)